pkg/controller/migmigration/restore.go

```diff
@@ -184,6 +184,12 @@ func (t Task) hasRestoreCompleted(restore *velero.Restore) (bool, []string) {
 				restore.Name))
 	case velero.RestorePhaseFailedValidation:
 		reasons = restore.Status.ValidationErrors
+		reasons = append(
+			reasons,
+			fmt.Sprintf(
+				"Restore: %s/%s validation failed.",
+				restore.Namespace,
+				restore.Name))
```

Comment:
While troubleshooting, I noticed the restore information was not being added to `Errors` for validation failures.  This really helps with investigation.